src/tests/daos_mw_fi.c

```diff
@@ -0,0 +1,264 @@
+/**
+ * (C) Copyright 2020-2023 Intel Corporation.
```

Comment:
I never know with these, but is this correct?
```suggestion
 * (C) Copyright 2023 Intel Corporation.
```

Reply:
yes indeed .. copy paste